src/bin/pgcopydb/pgsql.c

```diff
@@ -1435,15 +1435,15 @@ pgsql_set_search_path(PGSQL *pgsql, char *search_path, bool local)
 bool
 pgsql_prepend_search_path(PGSQL *pgsql, const char *namespace)
 {
-	char search_path[BUFSIZE] = { 0 };
+	char searchPath[BUFSIZE] = { 0 };
```

Comment:
Again, is that really a `cppcheck` thing?

Reply:
yes, `shadowed function name` warning.